site/en/guides/porting/set-up-the-build-environment.md

```diff
@@ -154,39 +175,11 @@ include $(top_srcdir)/examples/platforms/efr32/Makefile.platform.am
 endif
 ```
 
-### Subdirectory configuration
-
-Modify [`/openthread/examples/platforms/Makefile.am`](https://github.com/openthread/openthread/blob/main/examples/platforms/Makefile.platform.am)
-to configure the package subdirectories for the new platform example.
-
-Add the platform subdirectory name in the list for `make dist`, in alphabetical
-order:
-
-```
-# Always package (e.g. for 'make dist') these subdirectories.
- 
-DIST_SUBDIRS                           = \
-    cc2538                               \
-    cc2650                               \
-    efr32                                \
-    nrf52840                             \
-    simulation                           \
-    utils                                \
-    $(NULL)
-```
-
-Append platform subdirectory name to the `SUBDIRS` variable:
-
-```
-# Always build (e.g. for 'make all') these subdirectories.
- 
-if OPENTHREAD_EXAMPLES_EFR32
-    SUBDIRS                           += efr32
-endif
-```
 
 ### Toolchain startup code
 
+> *TODO*  Rewrite for CMake. Might not even be needed
```

Comment:
I'm not entirely sure what to do with this section. It really depends on how the rest of the guide is structured